addons/mail/static/src/core/web/activity_service.js

```diff
@@ -2,25 +2,31 @@
 
 import { _t } from "@web/core/l10n/translation";
 import { registry } from "@web/core/registry";
-import { browser } from "@web/core/browser/browser";
 
 export class ActivityService {
     /**
      * @param {import("@web/env").OdooEnv} env
      * @param {Partial<import("services").Services>} services
      */
     constructor(env, services) {
-        try {
-            // useful for synchronizing activity data between multiple tabs
-            this.broadcastChannel = new browser.BroadcastChannel("mail.activity.channel");
-            this.broadcastChannel.onmessage = this._onBroadcastChannelMessage.bind(this);
-        } catch {
-            // BroadcastChannel API is not supported (e.g. Safari < 15.4), so disabling it.
-            this.broadcastChannel = null;
-        }
         this.env = env;
         this.store = services["mail.store"];
         this.orm = services.orm;
+        this.multiTab = services["multi_tab"];
+        this.multiTab.bus.addEventListener("mail.activity.insert", ({ detail }) => {
+            this.store.Activity.insert(detail, { broadcast: false });
+        });
```

Comment:
Not really in scope of your PR, but I think `multiTab.bus.addEventListener()` kinda sucks: it's a pretty long LOC for just defining a listener. Would be better with just `multiTab.on()`

Reply:
Added a function to the service to abstract/shorten this usage.

Reply:
Finally, I removed this change and will eventually introduce it in a separated pull request.